decidim-core/app/models/decidim/user_group.rb

```diff
@@ -107,6 +107,11 @@ def accepted_users
       accepted_memberships.map(&:user)
     end
 
+    # Accepts conversation if all their users accepts a conversation?
+    def accepts_conversation?(_user)
```

Comment:
Yes. For the moment it returns true always for groups. That's because it is not been decided what should happen regarding all the "private messages status" for all the administrators of the group. According to @decidim/product, for the moment groups accepts conversation always. This method is here to allow to change that in the future.

But you are right, it should be described in the comment.